openupgrade_scripts/scripts/account/tests/data.py

```diff
@@ -0,0 +1,13 @@
+env = locals().get("env")
+# call sending wizard on some moves asynchronously
+action = (
+    env["account.move"]
+    .search([("move_type", "=", "out_invoice")])
```

Comment:
It seems invoice have to be posted before calling `action_send_and_print`.

Reply:
yes I'm still busy with this part

Reply:
https://github.com/ForgeFlow/OpenUpgrade/pull/26

there might be more to come, I'm still not done reviewing this. Feel free to rebase -i meanwhile

Reply:
Rebased :)